hack/multi-node/bootkube-up

```diff
@@ -10,7 +10,7 @@ fi
 SELF_HOST_ETCD=${SELF_HOST_ETCD:-false}
 if [ ${SELF_HOST_ETCD} = "true" ]; then
     etcd_render_flags="--etcd-servers=http://10.3.0.15:2379 --storage-backend=etcd3 --experimental-self-hosted-etcd"
-    etcd_start_flags="--etcd-server=http://172.17.4.101:12379,http://10.3.0.15:2379 --experimental-self-hosted-etcd"
+    etcd_start_flags="--etcd-server=http://172.17.4.101:12379 --experimental-self-hosted-etcd"
```

Comment:
See my explanation in https://github.com/kubernetes-incubator/bootkube/pull/259#issuecomment-272307260